### ISSUE TITLE
Create sweeps tests for reduce/fused/matmul based on aten traces

### ISSUE THREAD

**Issue by bbradelTT** [tenstorrent/tt-metal#15266]

**Comment by bbradelTT** [comment]
New sweeps merged.

Sweeps for backwards softmax and backwards layer_norm should be added as part of creating those ops. They should be based on:
- https://github.com/tenstorrent/pytorch2.0_ttnn/blob/main/docs/operations/aten._softmax_backward_data.default.md and
- https://github.com/tenstorrent/pytorch2.0_ttnn/blob/main/docs/operations/aten.native_layer_norm_backward.default.md
